### ISSUE TITLE
ToolBuilder?

### ISSUE THREAD

**Issue by LinqLover** [hpi-swa/widgets#9]
Not sure if I missed any existing implementation, but provided that there is not yet any UiToolBuilder, I would find it quite useful. I'm not deep into the widgets implementation, but ideally, you could replace the default MorphicToolBuilder in Squeak with this tool builder to completely change the design of Squeak. This would also be a nice demonstration of the adjustability of Squeak.

**Comment by marceltaeumel** [comment]
Yes, that's possible. However, one would also need a mapping from Squeak's changed/update mechanism to [signals](https://www.github.com/hpi-swa/signals).

A nice benchmark would be the System Browser, Object Inspector, and Debugger.
